profiles/README.md

```diff
@@ -10,13 +10,53 @@ Specifically, we include:
 2. Processed data for each Cell Painting plate (for several "data levels")
 3. Instructions on how to reproduce the profiling pipeline
 
-## Workflow
+## Workflow 
 
-![Cytominer workflow](media/cytominer_workflow.png)
+![Cytominer workflow](media/cytominer_workflow_new.png)
```

Comment:
please overwrite the file `cytominer_workflow.png` instead of creating a new file.

Github version control will help us to recover the old one in the unlikely event we ever need to revert back.